src/geode/io/model/svg_input.cpp

```diff
@@ -36,6 +36,7 @@
 
 #include <geode/mesh/builder/edged_curve_builder.h>
 #include <geode/mesh/builder/point_set_builder.h>
+#include <geode/mesh/core/edged_curve.h>
```

Comment:
why?

Reply:
It is needed for compilation, but I think previously it was included in another included header file (<geode/mesh/builder/edged_curve_builder.h>?), and not anymore

Reply:
or geode/model/mixin/line.h ?